javascriptv3/example_code/personalize/README.md

```diff
@@ -0,0 +1,81 @@
+#  Amazon Personalize JavaScript SDK v3 code examples
```

Comment:
# Amazon Personalize code examples for SDK for JavaScript (v3)

(per the template: https://github.com/awsdocs/aws-doc-sdk-examples/wiki/Service-level-README-template)